### PR TITLE
Translation to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by caiorrs**
Hey, I have translated it to my language =D

### CODE REVIEW

**Comment by k0shk0sh** [comment]
@caiorrs dude you are awesome, thank you very much :) one thing I noticed is that you have an old pull or FastHub which was using `buildtools-alpha3`, could you please pull again & see if there are some strings left behind and also stash out the changes you did in `build.gradle` since it isn't really needed :)

File: build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-alpha2'
+        classpath 'com.android.tools.build:gradle:3.0.0-alpha3'
```
Comment (k0shk0sh) [inline build.gradle]: please update your pull to FastHub latest and stash out this file :)

Comment (caiorrs) [inline build.gradle]: oh, hey, how do I do that? I'm kinda new using github+AS. I have forked your repo, then I used my fork on AS to commit and push. And then I made the pull request using the github website. How do I update the project?

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -0,0 +1,478 @@
+<resources>
+    <string name="app_name" translatable="false">FastHub</string>
+    <string name="app_name_full" translatable="false">FastHub for Android</string>
```
Comment (caiorrs) [inline app/src/main/res/values-pt-rBR/strings.xml]: hey there, what should I do with the translatable="false" strings? delete them from my file or should I translate them?

Comment (yakov116) [inline app/src/main/res/values-pt-rBR/strings.xml]: @caiorrs delete them if you pull the latest we have sorted them to be all on the top.

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: build.gradle (hunk 1 above)
Comment (yakov116) [inline build.gradle]: @caiorrs just change that field to a 3

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Comment (yakov116) [inline build.gradle]: @caiorrs ~~just change that field to a 3 
To alpha2 to alpha3.
If you cant figure it out I will fix it later. But please pull the latest changes. ~~

On second thought can you join us on slack and I will guide you?   

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: app/src/main/res/values-pt-rBR/strings.xml (hunk 2 above)
Comment (yakov116) [inline app/src/main/res/values-pt-rBR/strings.xml]: @caiorrs can you join us on slack?

Comment (caiorrs) [inline app/src/main/res/values-pt-rBR/strings.xml]: sure, whats the address?

Comment (k0shk0sh) [inline app/src/main/res/values-pt-rBR/strings.xml]: @caiorrs it's in readme (first top left image)

_Sent from my Htc m8 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Comment (yakov116) [inline app/src/main/res/values-pt-rBR/strings.xml]: @caiorrs http://rebrand.ly/fasthub

**Pull request closed without merging**